examples/gettingStarted/scheduledMessages.ts

```diff
@@ -0,0 +1,104 @@
+
+// Define connection string and related Service Bus entity names here
+const connectionString = "";
+const queueName = "";
+
+let ns: Namespace;
+
+const listOfScientists = [
+  { name: "Einstein", firstName: "Albert" },
+  { name: "Heisenberg", firstName: "Werner" },
+  { name: "Curie", firstName: "Marie" },
+  { name: "Hawking", firstName: "Steven" },
+  { name: "Newton", firstName: "Isaac" },
+  { name: "Bohr", firstName: "Niels" },
+  { name: "Faraday", firstName: "Michael" },
+  { name: "Galilei", firstName: "Galileo" },
+  { name: "Kepler", firstName: "Johannes" },
+  { name: "Kopernikus", firstName: "Nikolaus" }
+];
+
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(connectionString);
+  try {
+    await sendScheduledMessages();
+
+    await receiveMessages();
+  } finally {
+    await ns.close();
+  }
+}
+
+// Scheduling messages to be sent after 30 seconds from now
+async function sendScheduledMessages(): Promise<void> {
+  // If using Topics, use createTopicClient to send to a topic
+  const client = ns.createQueueClient(queueName);
+
+  const messages: SendableMessageInfo[] = listOfScientists.map((scientist) => ({
+    body: `${scientist.firstName} ${scientist.name}`,
+    label: "Scientist"
+  }));
+
+  const timeNowUtc = new Date(Date.now());
+  const scheduledEnqueueTimeUtc = new Date(Date.now() + 10000);
+  console.log(`>>>> Time now in UTC: ${timeNowUtc}`);
+  console.log(`>>>> Messages will appear in Service Bus at UTC: ${scheduledEnqueueTimeUtc}`);
+
+  await client.scheduleMessages(scheduledEnqueueTimeUtc, messages);
+  for (let index = 0; index < messages.length; index++) {
+    await client.scheduleMessage(scheduledEnqueueTimeUtc, messages[index]);
+    console.log(`Sent: ${messages[index].body} - ${messages[index].label}`);
+  }
+}
+
+async function receiveMessages(): Promise<void> {
+  // If using Topics, use createSubscriptionClient to receive from a topic subscription
+  const client = ns.createQueueClient(queueName);
+
+  const onMessage: OnMessage = async (brokeredMessage: ServiceBusMessage) => {
+    numOfMessagesReceived++;
+    console.log(`Received message: ${brokeredMessage.body} - ${brokeredMessage.label}`);
+
+    await brokeredMessage.complete();
+  };
+  const onError: OnError = (err: MessagingError | Error) => {
+    console.log("Error occurred: ", err);
+  };
+
+  let numOfMessagesReceived = 0;
+  const receiveListenerFirst = client.receive(onMessage, onError);
+  await delay(5000);
+  await receiveListenerFirst.stop();
+  console.log(
+    `\nStarted looking up immediately while having receive listener wait for 5 seconds. Found ${numOfMessagesReceived} messages.\n`
+  );
+
+  await delay(5000);
+
+  const receiveListenerSecond = client.receive(onMessage, onError);
+  await delay(5000);
+  await receiveListenerSecond.stop();
+  console.log(
+    `\nStarted looking up after 10 seconds while having receive listener wait, received ${numOfMessagesReceived} messages.`
+  );
```

Comment:
By this time, we have waited for a total of 15 seconds not 10.

Reply:
Technically, lookup started after 10 seconds. The messages start getting retrieved/displayed at 10 seconds. Lookup finished at 15 seconds

Reply:
I am guessing the intention of having the 2 receivers is to show that by the time the first message is printed, no messages were received. By the time the second one was printed, all (or atleast non zero) messages were received

When the user runs the sample and sees the messages, the point of reference they have is the scheduling of the messages.
The fact that we have 2 listeners or how much they waited is inconsequential to them until they go line by line in the code.

The first message is printed 5 seconds after scheduling the messages.
The second message is printed 15 seconds after scheduling the messages.

I believe it would be simpler if the messages used the scheduling of messages as point of reference.

Reply:
Yes, agree with using scheduled time as reference, which is 10 seconds. So mentioning 15 seconds may not seem relevant.
Updated initial message to indicate the 10 second scheduling delay that the sample is using.